vault/blog.community.2021.new-user-survey.md

```diff
@@ -0,0 +1,173 @@
+
+> The percentages are in relation to the 481 respondents in **Other**
+
+- Student/Learning/Class/Academia Notes (4%)
+- Journaling (4%)
+- Content Creation and Non-fiction/Fiction Writing (3%)
+- Project / Software Documentation (2%)
+- System/Product Configuration and Design (2%)
+- CRM (1%)
+- Second Brain (1%)
+- Gaming Notes (D&D, RPGs, General) (1%)
+
+There were some smaller groups that stood out in the **Other** category, having one to three respondents:
+
+- Scripted/Automated documentation creation
+- Tracking, Reading, and Taking Notes on Books
+- Recipes / Cookbook
+- Drafting GitHub issues/MRs
+- Event planning
+- Presentations
+
+## Are users coming from any existing tools?
+
+![Bar graph displaying different tools used before or with Dendron](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-previous-tools.png)
+
+New users were able to select all that applied. More users said they were coming from **No Previous Tool** (26%) than users who found themselves looking at Dendron after something else, with **OneNote** (25%) coming in a close second.
+
+Approximations of remaining previous (or parallel) tools:
+
+- **Notion** (23%)
+- **Google Keep** (19%)
+- **Obsidian** (18%)
+- **Evernote** (14%)
+- **Roam** (5%)
+- **Logseq** (5%)
+- **Foam** (5%)
+
+A total of 378 (15%) respondents provided additional information under **Other**, providing a total of over **70 different tools and workflows.** Here are the top **Other** tools listed.
+
+> The percentages are in relation to the 378 respondents in **Other**
+
+- Joplin (9%)
+- org-mode / org-roam / emacs (6%)
+- Bear (6%)
+- Apple Notes (4%)
+- Plain Markdown or text files (4%)
+- Trello / Confluence / Jira (4%)
+- Typora (3%)
+- Physical notes (2%)
+
+## Do users have plans on publishing their notes?
+
+![Bar graph displaying levels of interest in publishing with Dendron](https://org-dendron-public-assets.s3.amazonaws.com/images/2021-survey-publishing-interest.png)
+
+New users were only able to select a single response, if they were to respond to this question at all. A total of **2305 users responded.** A majority of users were **Curious** (39%) about publishing, or otherwise already saying **Yes** (36%) to planning publishing. The remaining respondents said **No** (25%) and were not interested in Dendron's publishing capabilities.
+
+## Takeaways
+
+Users are drawn to Dendron for Personal Knowledge Management (PKM), task management, research, and more.
+
+With that in mind, Dendron and the Dendron community have been introducing more features and enhancements that continue to develop:
+
+- [Task Notes](https://wiki.dendron.so/notes/SEASewZSteDK7ry1AshNG/)
+  - [Contribute to the GitHub Discussion](https://github.com/dendronhq/dendron/discussions/1358)
+- [Note Traits](https://wiki.dendron.so/notes/bdZhT3nF8Yz3WDzKp7hqh/)
```

Comment:
Should we link to the quickstart or the API instead of this note? Or add a bit of summary to this note since it's rather empty.